api/tests/conftest.py

```diff
@@ -3,7 +3,7 @@
 from fastapi.testclient import TestClient
 
 from database.database import Database
-from feeds_gen.main import app as application
+from main import app as application
```

Comment:
This is the application used, added here to ensure tests are aligned with running applications in a real environment.